SDK.ipynb

```diff
@@ -78,9 +91,21 @@
    },
    "outputs": [],
    "source": [
-    "import globus_sdk\n",
+    "# fill this line in with the code that you got\n",
+    "auth_code = \"YOUR CODE HERE\"\n",
+    "\n",
+    "# and exchange it for a response object containing your token(s)\n",
+    "token_response = native_auth_client.oauth2_exchange_code_for_tokens(auth_code)\n",
```

Comment:
We could also just print out a notice around where the link is that says it can only be used once and re-running the cell will generate a new one.

It's probably good information for people to have regardless of whether or not something goes wrong.